pkg/controller/ps/user_test.go

```diff
@@ -0,0 +1,156 @@
+package ps
+
+import (
+	"context"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	apiv1alpha1 "github.com/percona/percona-server-mysql-operator/api/v1alpha1"
+	"github.com/percona/percona-server-mysql-operator/pkg/secret"
+)
+
+func TestEnsureUserSecrets(t *testing.T) {
```

Comment:
Could you try to write those with Gingko?

Reply:
I will do it in another PR